### PR TITLE
Documentation for xloader

### PR DESCRIPTION
**Pull request by afs**

### CODE REVIEW

File: source/documentation/tdb/tdb-xloader.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+is stability and reliability for long running loading, running on modest and
+
+xloader is not a replacement for regular TDB1 and TDB2 loaders.
+
+"tdb1.xloader" was called "tdbloader2" and has some improvements.
+
+It is not as fast as other TDB loaders on dataset where the general loaders work
+on without encountering progressive slowdown.
+
+The xloaders for TDB1 and TDB2 are not identical. The TDB2 is more capable; it
```
Comment (rvesse) [inline source/documentation/tdb/tdb-xloader.md:15]: Awkward statement - "The TDB2 is more capable", maybe "TDB2 has more capabilities"

File: source/documentation/tdb/tdb-xloader.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+is based on the same design approach with further refinements to building the
+node table and to reduce the total amount of temporary file space used.
+
+The xloader does not run on MS Windows. It uses and external sort program from
```
Comment (rvesse) [inline source/documentation/tdb/tdb-xloader.md:19]: Typo - `and` should be `an`

File: source/documentation/tdb/tdb-xloader.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+### Running xloader
+
+`tdb2.xloader --loc DIRECTORY` FILE...
+
+or
+
+`tdb1.xloader --loc DIRECTORY` FILE...
+
+Additioally, there is an argument `--tmpdir` to use a different directory for
```
Comment (rvesse) [inline source/documentation/tdb/tdb-xloader.md:33]: Typo, missing `n` in `Additionally`

**Review by rvesse — COMMENTED** [review]
Documentation looks good, some minor typos

**Comment by rvesse** [comment]
Also is this going to be linked from elsewhere in the docs?

**Comment by afs** [comment]
Yes - there is some general revisions of TDB documentation to come but I've muddled up my working git repo and need to sort it out to branch off current main.

**Pull request merged**